fastcrypto/src/bls12381/mod.rs

```diff
@@ -350,19 +349,11 @@ impl Signature for BLS12381Signature {
 
 impl Default for BLS12381Signature {
     fn default() -> Self {
-        // TODO: improve this!
-        let ikm: [u8; 32] = [
-            0x93, 0xad, 0x7e, 0x65, 0xde, 0xad, 0x05, 0x2a, 0x08, 0x3a, 0x91, 0x0c, 0x8b, 0x72,
-            0x85, 0x91, 0x46, 0x4c, 0xca, 0x56, 0x60, 0x5b, 0xb0, 0x56, 0xed, 0xfe, 0x2b, 0x60,
-            0xa6, 0x3c, 0x48, 0x99,
-        ];
-
-        let sk = blst::SecretKey::key_gen(&ikm, &[]).unwrap();
+        let mut infinity: [u8; $sig_length] = [0; $sig_length];
+        infinity[0] = 0xc0;
```

Comment:
what is this value, is that based on the rfc?
let's add a comment explaining that

Reply:
> What is our stance if someone tries to verify against an infinity signature? we fail?

yes if it verifies against any list of pks it will fail.